charts/prometheus-druid-exporter/Chart.yaml

```diff
@@ -1,25 +1,25 @@
----
-apiVersion: v1
-appVersion: "v0.8.0"
+apiVersion: v2
```

Comment:
please add info in the upgrading section of the chart that this is a breaking change for helm 2 users.

please also remove all helm 2 commands from the readme.

Reply:
I did it